src/statement.ts

```diff
@@ -33,7 +33,7 @@ export function isStmt(a: any): a is Stmt {
       isForInStmt(a) ||
       isForOfStmt(a) ||
       isIfStmt(a) ||
-      isReturn(a) ||
+      isReturnStmt(a) ||
```

Comment:
Does this constitute a breaking change?

Reply:
No, I just renamed the method to be consistent

Reply:
unless these are exported in the npm package? then yes it would

Reply:
Yes, these are part of our public contract. It's a breaking change. Make sure to properly add that message to the PR and commit so that the version bumps appropriately